docs/organizer-guide/meetups/starting.rst

```diff
@@ -14,176 +13,186 @@ It contains a lot of the knowledge that we have put here:
 Write the Docs Worldview
 ------------------------
 
-Our worldview is important for starting a meetup.
-We aren't just another tech writing meetup,
-but hope to be welcoming to the vast parts of the tech industry that care about documentation.
+Yes, Write the Docs has a worldview.
+We include everyone who is interested in excellent documentation: tech writers, developers, engineers, QA, product managers, support engineers and more.
```

Comment:
We should probably mention devrel here, as I want to include them more in our universe.